english/hide/hide-chk.md

```diff
@@ -0,0 +1,3 @@
+you suspect the marshamllows are evil. they would like to snatch your lighter rope and fork and tie you the stake poke and burn you.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.